docs/migration-guides/qiskit-runtime.mdx

```diff
@@ -17,24 +17,13 @@ examples to help you migrate code from the legacy `backend.run` interface to use
+The `qiskit-ibm-runtime` package provides cloud access to the IBM&reg; quantum processing units  (QPUs) through the primitives interface. The `backend.run` interface coexisted with the original (V1) primitives model as the dedicated “direct hardware access” entry point.  With the introduction of the V2 primitives interface, the new `SamplerV2` class fulfills that role. Consequentially, `backend.run` and `qiskit-ibm-provider`, which only exposed the `backend.run` interface, are no longer supported.
 
 The Qiskit Runtime primitives implement the reference Sampler V2 and Estimator V2 interfaces found in `qiskit.primitives`, and enable capabilities not available with the legacy `backend.run` interface. These capabilities include application of advanced processing techniques for error suppression and mitigation in Estimator, the ability to efficiently sweep between arrays of parameter value sets or observables in both Sampler and Estimator, and access to the new local testing mode. Additionally, Qiskit Runtime lets users run iterative algorithm circuits back to back (session mode) or in collections of circuits without having to re-queue each job (batch mode). This results in more efficient quantum processor use and reduces the time spent running complex computations.
 
 ## Why migrate to Qiskit Runtime primitives?
 
-Similar to the early days of classical computers, when developers had to manipulate CPU registers directly,
-the early interface to QPUs simply returned the raw data coming out of the control electronics.
-This was not a huge issue when QPUs lived in labs and only allowed direct access by researchers.
-Acknowledging that most developers would not and should not be familiar with distilling such raw data into
-0s and 1s, IBM introduced `backend.run`, a first abstraction for accessing QPUs in the cloud. This allowed developers
-to operate on a familiar data format and focus on the bigger picture.
-
-As access to QPUs became more widespread, and with more quantum algorithms being developed,
-again the need for a higher-level abstraction emerged. In response, IBM introduced
-the Qiskit [primitives interface](/guides/primitives), which are optimized for two core tasks in quantum algorithm development:
-expectation value estimation (`Estimator`) and circuit sampling (`Sampler`). The goal is once
-again to help developers to focus more on innovation and less on data conversion.
+The `backend.run` interface was the primary access point to quantum hardware and simulators offered by IBM, as well as third-party providers that integrate with Qiskit. The [evolution of user access patterns](/guides/primitives#qpu-access-patterns) for quantum processing units (QPUs) led to the Qiskit primitives (`Sampler` and `Estimator`) superseding this interface.
```

Comment:
```suggestion
The `backend.run` interface was the primary access point to quantum hardware and simulators offered by IBM, as well as third-party providers that integrate with Qiskit. The [evolution of user access patterns](/guides/primitives#why-did-qiskit-introduce-primitives) for quantum processing units (QPUs) led to the Qiskit primitives (`Sampler` and `Estimator`) superseding this interface.
```

Reply:
(/guides/primitives#qpu-access-patterns) is the right link because I have an anchor there. I don't understand why the anchor doesn't show up when you hover, so I know it's confusing.  But it's better to use the anchor in case we change the title.